src/autoTune.ml

```diff
@@ -218,17 +218,21 @@ let enableAnalyses anas =
 (*does not consider dynamic calls!*)
 
 let notNeccessaryThreadAnalyses = ["race"; "deadlock"; "maylocks"; "symb_locks"; "thread"; "threadid"; "threadJoins"; "threadreturn"; "mhp"; "region"; "pthreadMutexType"]
-let reduceThreadAnalyses () =
+let notNeccessaryRaceAnalyses = ["race"; "symb_locks"; "region"]
+let reduceAnalyses () =
   let isThreadCreate (desc: LibraryDesc.t) args =
     match desc.special args with
     | LibraryDesc.ThreadCreate _ -> true
     | _ -> LibraryDesc.Accesses.find_kind desc.accs Spawn args <> []
   in
   let hasThreadCreate = hasFunction isThreadCreate in
-  if not @@ hasThreadCreate then (
-    Logs.info "no thread creation -> disabling thread analyses \"%s\"" (String.concat ", " notNeccessaryThreadAnalyses);
-    disableAnalyses notNeccessaryThreadAnalyses;
-  )
+  let hasDataRaceSpec = List.exists (SvcompSpec.equals SvcompSpec.NoDataRace) (Svcomp.Specification.of_option ()) in
```

Comment:
This could be `List.mem` (`BatList.mem_cmp` to be more precise), but that needs `compare` instead of `equal` (could also be derived).